ts/nni_manager/training_service/remote_machine/shellExecutor.ts

```diff
@@ -169,6 +169,16 @@ class ShellExecutor {
         return this.tempPath;
     }
 
+    public async getHomePath(): Promise<string> {
```

Comment:
The name looks like it returns `/home/zzz`, but it actually returns cwd?

Reply:
Yes, `cwd` is we need, I will change it to `getCurrentPath`.